tests/plugins/contextmenu/manual/hide.html

```diff
@@ -0,0 +1,27 @@
+<style>
+	.marker {
+		width: 50px;
+		height: 50px;
+		background: blue;
+		margin: 5px;
+	}
+</style>
+<textarea id="editor"><div style="width:100%;height:200px;"></div></textarea>
```

Comment:
What is the purpose of the empty `div` inside editor? It renders unnecessary border on click in IE 8.

Reply:
It fills the editor, so it is possible to open context menu.